_data/simple-icons.json

```diff
@@ -3304,6 +3304,11 @@
             "hex": "F01F7A",
             "source": "https://codecov.io"
         },
+        {
+            "title": "CodeCrafters",
+            "hex": "3475E8",
```

Comment:
@simple-icons/pr-reviewers are we okay with this color?

Reply:
It's tough because they seem to use more magenta/violet on the website despite the color here being the correctly calculated gradient of the logo. It might be worth opening an issue on https://github.com/codecrafters-io/frontend or pinging someone who works at the company?

Reply:
If you would like I can contact someone from the company to find out an answer for us

Reply:
That would be great!

Reply:
I sent an email to one of the founders, hopefully we can hear something back soon 🚀

Reply:
![image](https://github.com/user-attachments/assets/3319a891-d6d8-462c-aa22-c507d77fa8ca)

Reply:
Responded here: https://github.com/simple-icons/simple-icons/pull/11252/files#r1760910150. Thanks everyone!